contrib/completion/bash/docker

```diff
@@ -259,7 +259,7 @@ _docker_build() {
 
 	case "$cur" in
 		-*)
-			COMPREPLY=( $( compgen -W "--force-rm --no-cache --quiet -q --rm --tag -t" -- "$cur" ) )
+			COMPREPLY=( $( compgen -W "-f --force-rm --no-cache --quiet -q --rm --tag -t" -- "$cur" ) )
```

Comment:
This needs `--file` as well, and we need to add `--file|-f)` in the `case` block above this, preferably completing to `_filedir`.